BeginnerGuide.md

```diff
@@ -0,0 +1,124 @@
+# **ABSOLUTE BEGINNERS GUIDE**
+
+## **OWASP WRONGSECRETS**
+
+Basically, [*WrongSecrets*](https://owasp.org/www-project-wrongsecrets/) is an application which teaches how to not store secrets by offering challenges to the user, which helps the user to Self-reflect and correct those mistakes.
+
+---
+
+### *What are the things you need to know?*
+
+Don’t worry too much if you do not understand the code just reach out to the [*community*](https://app.slack.com/client/T04T40NHX/C02KQ7D9XHR), they’ll help you for sure.
+Here is some basic Glossary.
+
+1. **Secrets:**  
+   [*Secrets*](https://delinea.com/what-is/application-secrets) refer to a private piece of information that acts as a key to unlock protected resources or sensitive information in tools, applications, containers, and cloud-native environments.
```

Comment:
@bendehaan It would be great if everything related to the project would be centrlized because couldn't get my hands on cheat sheet in my beginner days, i googled it myself and found it out the hard way, i can link the cheat sheet to it.
Like for "More information Please refer the OWASP Cheat sheet." 

> Any particular reason for including this specific resource? Are there any OWASP resources available (e.g., on the wiki or in the cheat sheet)?